docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -55,20 +55,31 @@ Node.js::
 -- 
 [source,javascript]
 ----
+class Employee {
 
+    const surname;
+
+    constructor(surname) {
+        this.surname = surname;
+    }
+
+    toString() {
+        return "Employee: " + this.surname;
+    }
+}
```

Comment:
This is not valid Node.js:

![image](https://user-images.githubusercontent.com/32355782/187666111-3a42dc53-cc08-4a15-8c4f-53b289d41f62.png)